jsonschema-core.xml

```diff
@@ -1228,7 +1228,14 @@
                         The "$schema" keyword SHOULD be used in the document root schema object,
                         and MAY be used in the root schema objects of embedded schema resources.
                         It MUST NOT appear in non-resource root schema objects.  If absent from
-                        the document root schema, the resulting behavior is implementation-defined.
+                        the document root schema, the resulting behavior is implementation-defined,
+                        but MUST fall within the following options:
+                        <ul>
+                            <li>Refuse to process the schema, as with unsupported required vocabularies
+                            <li>Assume a specific, documented meta-schema
```

Comment:
I'll just point out that under this language, it is still valid behavior to proceed with validation and consider all instances *valid*, or all instances *invalid* (because I can document my specific documented meta-schema is `{}` or an equivalent of `{"not": {}}`).

I don't see that as an issue personally, I think there are reasonable reasons that may make sense, but will mention it explicitly since I know there was argument about those cases and whether they're desired behavior or not.

Reply:
Yeah, the main issue is that we can't, in the core spec, restrict the options to just the standard meta-schemas.  There are perfectly good reasons to default to something non-standard (e.g. OpenAPI's dialect).  There are endless ways to write always-passing and always-failing schemas, so trying to forbid those feels like a bit of a lost cause.

Requiring that one document the meta-schema should at least highlight the possibility, and I suspect people would feel a bit silly documenting `{}` as their default meta-schema.  But at least it would be clear.